.gitignore

```
@@ -2,4 +2,4 @@
 /be/compendium/compendium/.env
 /be/compendium/db/__pycache__
 /be/compendium/db/migrations/__pycache__
-/be/venv
+/be/.venv
```

```suggestion
/be/venv
/be/.venv
```
please still include these two as this might differ depending on the user's environment

---

Done, thanks